lib/rubocop/cop/rails/use_time_zone.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for the use of Time.zone= method.
+      # Built on top of this nice Thoughtbot article (https://thoughtbot.com/blog/its-about-time-zones)
+      #
+      # @example
+      #   # bad
+      #   Time.zone = 'EST'
+      #
+      #   # good
+      #   Time.use_zone('EST', &block)
+      #
+      class UseTimeZone < Base
```

Comment:
Can you rename the cop?

```suggestion
      class TimeZoneAssignment < Base
```